ste/sender-appendBlob.go

```diff
@@ -139,11 +141,21 @@ func (s *appendBlobSenderBase) Prologue(ps common.PrologueState) (destinationMod
 	s.headersToApply.ContentType = ps.GetInferredContentType(s.jptm)
 
 	destinationModified = true
-	_, err := s.destAppendBlobURL.Create(s.jptm.Context(), s.headersToApply, s.metadataToApply, azblob.BlobAccessConditions{})
-	if err != nil {
+	blobTags := s.blobTagsToApply
+	setTagsRequired := setTagsRequired(blobTags)
```

Comment:
>setTagsRequired [](start = 1, length = 15)

Readability: consider rename to something like separateSetTagRequired to make its intent clearer

Reply:
Done